src/js/server/schemaHandler.js

```diff
@@ -0,0 +1,53 @@
+/*
+
+    An extension of `gpii.express.handler` that adds JSON Schema headers to the outgoing response.
+
+    See this component's documentation for more details:
+
+    https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-1336/docs/handler.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-express");
+
+fluid.registerNamespace("gpii.schema.handler");
+
+// Send the appropriate headers and then let the underlying grade's `sendResponse` function take over.
+gpii.schema.handler.sendResponse = function (that, response, statusCode, body) {
+    if (!that.options.schemaKey || !that.options.schemaUrl) {
+        fluid.log("Your gpii.schema.handler is not configured correctly and cannot set the appropriate headers.");
+    }
+    else if (response.headersSent) {
+        fluid.log("Can't set headers, they have already been sent.");
+    }
+    else {
+        response.type("application/" + that.options.schemaKey + "+json; profile=\"" + that.options.schemaUrl + "\"");
```

Comment:
No escaping rules here? What characters are forbidden from appearing in a "schemaKey" by virtue of its use in a literal header here?